pkg/assets/create/creater.go

```diff
@@ -57,45 +55,47 @@ func EnsureManifestsCreated(ctx context.Context, manifestDir string, restConfig
 		options.StdErr = os.Stderr
 	}
 
-	// Default QPS in client (when not specified) is 5 requests/per second
-	// This specifies the interval between "create-all-resources", no need to make this configurable.
-	interval := 200 * time.Millisecond
-
-	// Retry creation until no errors are returned or the timeout is hit.
 	var (
 		lastCreateError      error
-		retryCount           int
 		mapper               meta.RESTMapper
 		needDiscoveryRefresh bool = true
 	)
-	err = wait.PollImmediateUntil(interval, func() (bool, error) {
-		retryCount++
+	for retryCount := 1; ; retryCount++ {
+		if retryCount > 1 {
+			select {
+			case <-ctx.Done():
+				if lastCreateError == nil {
```

Comment:
nit: completely ignorable ;)
```go
if lastCreateError != nil {
    return lastCreateError
}
return ctx.Err()
```
IMO that is more `Return the last observed set of errors from the create process instead of timeout error.` ...

Reply:
> nit: completely ignorable ;)
>
> ```go
> if lastCreateError != nil {
>     return lastCreateError
> }
> return ctx.Err()
> ```

Isn't that just reversing the order (but not changing the behavior) of my current:

```go
if lastCreateError == nil {
  return ctx.Err()
}
return lastCreateError
```

?  I'm fine updating if you like, but when I have both `if` and `else` cases (with a flattened `else` here) with short bodies, I prefer the positive `==` to the negative `!=` so you don't have a double-negative in your head for the `else` case.